awswrangler/athena/_read.py

```diff
@@ -979,3 +1150,93 @@ def read_sql_table(
         s3_additional_kwargs=s3_additional_kwargs,
         pyarrow_additional_kwargs=pyarrow_additional_kwargs,
     )
+
+
+@apply_configs
+def unload(
+    sql: str,
+    path: str,
+    database: str,
+    file_format: str = "PARQUET",
+    compression: Optional[str] = None,
+    field_delimiter: Optional[str] = None,
+    partitioned_by: Optional[List[str]] = None,
+    workgroup: Optional[str] = None,
+    encryption: Optional[str] = None,
+    kms_key: Optional[str] = None,
+    boto3_session: Optional[boto3.Session] = None,
+    data_source: Optional[str] = None,
+    params: Optional[Dict[str, Any]] = None,
+) -> _QueryMetadata:
```

Comment:
This is saying that a _QueryMetadata type is returned but the docs (line 1210) talks about an iterator of dataframes...

Reply:
Thanks